plugins/woocommerce/i18n/phone.php

```diff
@@ -261,6 +261,7 @@
 	'AW' => '+297',
 	'IN' => '+91',
 	'AX' => '+35818',
+	'XK' => '+383',
```

Comment:
There is an existing empty string for `XK` on [line 161](https://github.com/woocommerce/woocommerce/blob/a28a24c76c2d2ba8bc0e05ff284832e83e0af68a/plugins/woocommerce/i18n/phone.php#L161). Rather than add a duplicate, could we update the code there?